sinks/gcm/driver.go

```diff
@@ -191,7 +214,8 @@ type timeseriesDescriptor struct {
 type point struct {
 	Start       time.Time `json:"start,omitempty"`
 	End         time.Time `json:"end,omitempty"`
-	DoubleValue float64   `json:"doubleValue"`
+	DoubleValue *float64  `json:"doubleValue,omitempty"`
```

Comment:
we can't do omitempty unfortunately since zero is a valid value :( We need two different structs one for each type.